spec/rubocop/config_loader_spec.rb

```diff
@@ -149,6 +149,40 @@
       end
     end
 
+    context 'when the project is in a subdirectory of $HOME' do
+      let(:file_path) { File.expand_path('~/dir/.rubocop.yml') }
+
+      before do
+        Dir.mkdir(File.expand_path('~/dir'))
+
+        create_file(file_path, <<~YAML)
```

Comment:
The test here should look more like the nested test below so it would fail correctly if we had merged this code.